src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindOneAndUpdateNoIndexIntegrationTest.java

```diff
@@ -126,8 +126,7 @@ public void byIdAfterUpdate() {
                         "name": "Joe",
                         "age": 42,
                         "enabled": true,
-                        "value": -1,
-                        "$vector" : [ 0.5, -0.25 ]
```

Comment:
Simplifying the test since $vector not really needed for test (but new default projection would drop)